vaadin-testbench-core/src/test/java/com/vaadin/testbench/unit/UIUnit4BaseClassTest.java

```diff
@@ -57,13 +57,17 @@ public void extendingBaseClass_runTest_vaadinInstancesAvailable() {
         public void extendingBaseClass_runTest_defaultRouteActive() {
             Assert.assertTrue(
                     "Expecting default route to be active, but was not",
-                    UI.getCurrent().getInternals().getActiveRouterTargetsChain()
-                            .get(0) instanceof Home);
+                    getCurrentView() instanceof Home);
         }
 
     }
 
     public static class DiscoverAllRoutesTest extends UIUnit4Test {
+        @Override
+        protected String scanPackage() {
+            return "com.example.base";
+        }
```

Comment:
The test becomes an upkeep test if it is for all as we probably want to add new routes for different reasons. But I'll update them for this round.

Reply:
The test is there because base class has **null** as default for `scanPackage` method, but it may not be a good default.
Perhaps we can change it in another PR with a better value, because it is unlikely that developer needs to scan full classpath; test class package may be a better choice.